selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -886,36 +889,81 @@ prepareTestUserForMultiuserChe() {
         local cli_auth="--no-config --server http://localhost:8080/auth --user ${CHE_ADMIN_NAME} --password ${CHE_ADMIN_PASSWORD} --realm master"
         local response=$(docker exec -i $kc_container_id sh -c "keycloak/bin/kcadm.sh create users -r che -s username=${CHE_TESTUSER_NAME} -s enabled=true $cli_auth 2>&1")
         if [[ "$response" =~ "Created new user with id" ]]; then
-           NEW_USER_ID=$(echo "$response" | grep "Created new user with id" | sed -e "s#Created new user with id ##" | sed -e "s#'##g")
+           NEW_DEFAULT_USER_ID=$(echo "$response" | grep "Created new user with id" | sed -e "s#Created new user with id ##" | sed -e "s#'##g")
            # set test user's permanent password
            docker exec -i $kc_container_id sh -c "keycloak/bin/kcadm.sh set-password -r che --username ${CHE_TESTUSER_NAME} --new-password ${CHE_TESTUSER_PASSWORD} $cli_auth"
            # set email of test user to ${cheTestUserEmail}
-           docker exec -i $kc_container_id sh -c "keycloak/bin/kcadm.sh update users/${NEW_USER_ID} -r che --set email=${CHE_TESTUSER_EMAIL} $cli_auth"
+           docker exec -i $kc_container_id sh -c "keycloak/bin/kcadm.sh update users/${NEW_DEFAULT_USER_ID} -r che --set email=${CHE_TESTUSER_EMAIL} $cli_auth"
            # add realm role "user" test user
            docker exec -i $kc_container_id sh -c "keycloak/bin/kcadm.sh add-roles -r che --uusername ${CHE_TESTUSER_NAME} --rolename user $cli_auth"
            # add role "read-token" of client "broker" to test user
            docker exec -i $kc_container_id sh -c "keycloak/bin/kcadm.sh add-roles -r che --uusername ${CHE_TESTUSER_NAME} --cclientid broker --rolename read-token $cli_auth"
         else
            # set test user credentials to be equal to admin ones in case of problem with creation of user
-           echo -e "${RED}[WARN] There is a problem with creation of test user in Keycloak server: '${response}'.${NO_COLOUR}"
-           echo -e "Admin user will be used as a test user."
+           echo -e "${RED}[WARN] There is a problem with creation of default test user in Keycloak server: '${response}'.${NO_COLOUR}"
+           echo -e "Admin user will be used as default test user."
            CHE_TESTUSER_NAME=${CHE_ADMIN_NAME}
            CHE_TESTUSER_EMAIL=${CHE_ADMIN_EMAIL}
            CHE_TESTUSER_PASSWORD=${CHE_ADMIN_PASSWORD}
         fi
+        
+        # create second test user
```

Comment:
It is possible to use the same function to create both users?

Reply:
yes, it is possible

Reply:
see fixup in new commit https://github.com/eclipse/che/pull/8342/commits/0eb82c942060c7c2d5863d0daf9559514a4cae94